### DISCUSSION TITLE
Plus OptiTrack-Telemed package for 3D freehand ultrasound acquisition in 3D slicer

### DISCUSSION THREAD

**Discussion by kuleuvenUS** [PlusToolkit/PlusLib#802] [Q&A] [answered]
Hi Plus community!

With the biomechanics lab at KU Leuven (Belgium), we are looking to start our 3D freehand ultrasound measurements with the 3D slicer software. We found that the Plus toolkit is required in order to synchronize both the 3D motion capture system and the ultrasound system for acquiring the images. In our case, we are using the OptiTrack V120:trio and the ArtUS (Telemed) hardwares, respectively. 
I saw on the website that this working package for Plus **OptiTrack-Telemed-win32** was available upon request, however I didn’t find where I could do so, so that’s why I’m reaching out to this community.
Furthermore, I was wondering if this system only works on the Windows 7 and 8 systems as mentioned or if there might have been a more recent version working with windows 10 maybe? However, I saw there was a package for Telemed for Windows 10 but I’m assuming that might not be compatible with OptiTrack as well then (we only have the USB hardware licence and no Motive software licence)?

Thank you for your help!

**Comment by Sunderlandkyl** [comment]
Sure, I'll upload an OptiTrack-Telemed package for you. It should work on Windows 10 as well.

**Reply by kuleuvenUS** [reply]
I'm not familiar with GitHub and I haven't found the package in the downloads section on the website. Maybe I'm missing something where I can find the package?
Thanks!

**Reply by Sunderlandkyl** [reply]
I'll add a download link here. I just need to set up a new build for the package.

**Reply by Sunderlandkyl** [reply]
Try downloading the PlusApp-2.8.0.20191105-Telemed-Win32.exe from: http://perk-software.cs.queensu.ca/plus/packages/stable/, and let me know if you have any issues.

**Comment by ungi** [comment, marked as answer]
FYI, I use the same setup, Telemed and OptiTrack almost every day. Let me know you run into any practical problems and I can most probably help. My email address is ungi (at) queensu.ca
